runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/environment/InProcessEnvironmentFactory.java

```diff
@@ -81,6 +81,7 @@ private InProcessEnvironmentFactory(
   }
 
   @Override
+  @SuppressWarnings("FutureReturnValueIgnored")
```

Comment:
Minor: suggest inline document as to why - something along the lines of.

`// checked inline in an asynchronous task`

Reply:
Done; thanks for the suggestion.